node/testing.go

```diff
@@ -27,3 +34,41 @@ func TestNode(t *testing.T, tp Type, opts ...Option) *Node {
 	require.NoError(t, err)
 	return nd
 }
+
+// WithFaultMaker is used to enable test cases with invalid blocks
+func WithFaultMaker(faultHeight int64) Option {
```

Comment:
`node.WithFaultMaker` name does not explain what exactly is getting faulted. Add Header to the option name